inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -400,19 +401,19 @@ def refextract(obj, eng):
     references provided by the submitter, if any, then chooses the one
     that generated the most and attaches them to the workflow object.
 
-    Note:
-        We might want to compare the number of *matched* references instead.
-
     Args:
         obj: a workflow object.
         eng: a workflow engine.
 
     Returns:
         None
+
     """
     if 'references' in obj.data:
-        obj.log.info('Found references in metadata, extracting unextracted raw_refs')
-        obj.data['references'] = extract_references_from_raw_refs(obj.data['references'])
+        raw_references = extract_references_from_raw_refs(obj.data['references'])
```

Comment:
why is this called `raw_references`? I would expect the result to be a list of parsed, structured references after processing by refextract. Maybe `extracted_raw_references`?